stdlib/distutils/util.pyi

```diff
@@ -1,23 +1,44 @@
-from typing import Any, Callable, List, Mapping, Optional, Tuple
+from collections.abc import Callable, Container, Iterable, Mapping
+from typing import Any, Tuple
 
 def get_platform() -> str: ...
 def convert_path(pathname: str) -> str: ...
 def change_root(new_root: str, pathname: str) -> str: ...
 def check_environ() -> None: ...
 def subst_vars(s: str, local_vars: Mapping[str, str]) -> None: ...
-def split_quoted(s: str) -> List[str]: ...
+def split_quoted(s: str) -> list[str]: ...
 def execute(
-    func: Callable[..., None], args: Tuple[Any, ...], msg: Optional[str] = ..., verbose: bool = ..., dry_run: bool = ...
+    func: Callable[..., None], args: Tuple[Any, ...], msg: str | None = ..., verbose: bool = ..., dry_run: bool = ...
 ) -> None: ...
 def strtobool(val: str) -> bool: ...
 def byte_compile(
-    py_files: List[str],
+    py_files: list[str],
     optimize: int = ...,
     force: bool = ...,
-    prefix: Optional[str] = ...,
-    base_dir: Optional[str] = ...,
+    prefix: str | None = ...,
+    base_dir: str | None = ...,
     verbose: bool = ...,
     dry_run: bool = ...,
-    direct: Optional[bool] = ...,
+    direct: bool | None = ...,
 ) -> None: ...
 def rfc822_escape(header: str) -> str: ...
+def run_2to3(
+    files: Iterable[str],
+    fixer_names: Iterable[str] | None = ...,
+    options: Mapping[str, Any] | None = ...,
+    explicit: Container[str] | None = ...,
```

Comment:
This is apparently ignored.

Reply:
I noticed that too and was thinking about the options:

1. Just type it as `None`. This would notify a user that there is something strange going on.
2. Type it as above as that is what would be correct if this code was used.
3. Type it using `object` as technically that is correct.

Reply:
I can see a case for all three of these; perhaps at some point we can come up with a typeshed-wide convention. For now I suggest we add a comment so we can find this back if we ever decide to change the convention:
```suggestion
    explicit: Container[str] | None = ...,  # unused
```